back/src/chat/channel/channel.service.ts

```diff
@@ -231,6 +259,8 @@ export class ChannelService {
 					description: 'Something went terribly wrong.'
 				});
 			});
+		this.chatService.joinRoom(socketId, channel.id.toString());
```

Comment:
Maybe add some emits here to notify clients connected to the current channel that someone has joined the channel (so, in the fronted, we can display something like "hey, ${username} has joined the channel, say hello!!!`).